analyzer/evmverifier/sourcify/client.go

```diff
@@ -34,6 +33,15 @@ var sourcifyChains = map[common.ChainName]map[common.Runtime]string{
 	},
 }
 
+// Level of contract verification, as defined by Sourcify.
+// Keep values in sync with the `sourcify_level` postgres ENUM.
+type VerificationLevel string
```

Comment:
Is there no plan/need to expose this to the frontend?

Reply:
Oh :)
Added a new field to the relevant API endpoints now, thanks.